### ISSUE TITLE
Add blocked Cloudflare IPs

### ISSUE THREAD

**Issue by aivus** [zhovner/zaborona_help#178]
Some Cloudflare IPs are block due to https://nkrzi.gov.ua/index.php?r=site/index&pg=99&id=1870&language=uk

Particularly opencollective.com (proof: https://github.com/opencollective/opencollective/issues/3026)

It will be good to add this blocked addesses to the VPN:
```
➜ ~ nslookup opencollective.com
Server:		1.1.1.1
Address:	1.1.1.1#53

Non-authoritative answer:
Name:	opencollective.com
Address: 104.26.12.145
Name:	opencollective.com
Address: 104.26.13.145
Name:	opencollective.com
Address: 172.67.71.29
```

**Comment by toxi22** [comment]
Perhaps this is your provider blocking. I checked at home - it works without problems.
![2020-06-05_223244](https://user-images.githubusercontent.com/4610098/83915790-9c66fd80-a77c-11ea-8ace-3aa22e7fd9a4.png)

**Comment by aivus** [comment]
Probably it depends on the block list which each provider is using.

But like it's mentioned in original issue [Triolan](http://www.triolan.com/) is the one of the biggest ISP in Ukraine.

4th place across Ukraine based on the article on 2016:
http://en.mediasat.info/2016/03/01/broadband-internet-in-ukraine/

**Comment by toxi22** [comment]
Try to contact the technical support of the provider and ask them if they are blocking this address or is it just a failure?

**Comment by aivus** [comment]
@toxi22 I've contacted. The reason - the reason is in the first comment:
https://nkrzi.gov.ua/index.php?r=site/index&pg=99&id=1870&language=ru

**Comment by toxi22** [comment]
True, this IP is used for many sites, and not for one. Therefore, they blocked access to other sites that use the same IP address.
We will wait for a response from the server administrator if this site can be added to the VPN list. If he gives permission, the site will appear in the block bypass list.
